### PR TITLE
Add README for NGINX OSS & Plus receivers

### PR DESCRIPTION
**Pull request by dhurley**
### Proposed changes

Add README for NGINX OSS & Plus receivers

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [ ] I have read the [`CONTRIBUTING`](https://github.com/nginx/agent/blob/main/docs/CONTRIBUTING.md) document
- [ ] I have run ```make install-tools``` and have attached any dependency changes to this pull request
- [ ] If applicable, I have added tests that prove my fix is effective or that my feature works
- [ ] If applicable, I have checked that any relevant tests pass after adding my changes
- [ ] If applicable, I have updated any relevant documentation ([`README.md`](https://github.com/nginx/agent/blob/main/README.md))
- [ ] If applicable, I have tested my cross-platform changes on Ubuntu 22, Redhat 8, SUSE 15 and FreeBSD 13

### CODE REVIEW

File: internal/collector/nginxossreceiver/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# NGINX Receiver
+
+This receiver can fetch stats from a NGINX instance by using two different ways of scraping:
```
Comment (servusdei2018) [inline internal/collector/nginxossreceiver/README.md:3]: Perhaps reword this to "This receiver can fetch metrics from a NGINX instance via two sources:"

File: internal/collector/nginxossreceiver/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# NGINX Receiver
+
+This receiver can fetch stats from a NGINX instance by using two different ways of scraping:  
+* Scarping metrics from the `ngx_http_stub_status_module` module's `status` endpoint.
```
Comment (servusdei2018) [inline internal/collector/nginxossreceiver/README.md:4]: Typo "Scarping" -> "Scraping"

File: internal/collector/nginxplusreceiver/README.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# NGINX Plus Receiver
+
+This receiver can fetch stats from a NGINX Plus instance from the `ngx_http_api_module` module's `api` endpoint.
```
Comment (servusdei2018) [inline internal/collector/nginxplusreceiver/README.md:3]: To avoid two "from"s we can convert the second from to "via"

**Review by aphralG — APPROVED** [review]

**Review by servusdei2018 — APPROVED** [review]

**Review by RRashmit — APPROVED** [review]

**Review by sean-breen — APPROVED** [review]
